samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/LayerLists.razor

```diff
@@ -0,0 +1,15 @@
+﻿@page "/layerLists"
+<PageTitle>LayerLists</PageTitle>
+<h1>LayerLists</h1>
+
+<p class="instructions">
```

Comment:
Add an `<a>` tag that links to the JS API page for LayerLists. See other sample pages for example.

Reply:
corrected